examples/mdetr/vqa_eval.py

```diff
@@ -136,18 +160,20 @@ def main(args):
     else:
         checkpoint = torch.load(args.resume, map_location="cpu")
 
-    model_without_ddp.load_state_dict(checkpoint["model"])
+    model_without_ddp.load_state_dict(checkpoint["model"], strict=False)
```

Comment:
why not strict? how was it working till now?

Reply:
tagged you in the old model.py file with the explanation